app/Http/Controllers/Twill/BuildingClosureController.php

```diff
@@ -2,54 +2,76 @@
 
 namespace App\Http\Controllers\Twill;
 
+use A17\Twill\Services\Listings\Columns\Text;
+use A17\Twill\Services\Listings\Filters\BasicFilter;
+use A17\Twill\Services\Listings\Filters\TableFilters;
+use A17\Twill\Services\Listings\TableColumns;
 use App\Models\BuildingClosure;
+use Illuminate\Database\Eloquent\Builder;
 
-class BuildingClosureController extends \App\Http\Controllers\Twill\ModuleController
+class BuildingClosureController extends BaseController
 {
-    protected $moduleName = 'buildingClosures';
+    public function setUpController(): void
+    {
+        parent::setUpController();
+        $this->setModuleName('buildingClosures');
+        $this->setSearchColumns(['closure_copy']);
+        $this->setTitleColumnKey('presentType');
```

Comment:
The title column isn't sortable on this table, I think because this is a calculated field and not a DB field. I can't think of a better way to do this though. There's not another field that would naturally work as a title. We could try using the start date as the title? I think this PR is OK to merge as it is though.